Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift

```diff
@@ -540,6 +563,45 @@ private extension CardPresentPaymentStore {
         storage.deleteObject(storageAccount)
         storage.saveIfNeeded()
     }
+
+    func upsertChargeInBackground(readonlyCharge: WCPayCharge) {
```

Comment:
Not sure I understand the `Background` in the name. As far as I can see it all happens in the main thread

Reply:
Yep, you're right, I'd made this background at first then switched to sync.